src/client/hooks/useFetchTheme.ts

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react'
+
+import { listThemes } from '../../utils'
+
+export async function useFetchTheme(
+  setThemeKeys: React.Dispatch<React.SetStateAction<{}>>
+): Promise<void> {
+  const getThemes = async (): Promise<void> => {
+    const listOfThemes = await listThemes()
+    setThemeKeys(listOfThemes)
+  }
+
+  useEffect(() => {
+    getThemes().catch(e => console.log(e))
+  }, [])
```

Comment:
I believe the Effect hook is similar to a lifecycle method for a component. This doesn't seem to be a component, so not sure how useful `useEffect` is here. Try deleting lines 13-15.